### PR TITLE
Add documentation for MsgSlashValidator

### PR DESCRIPTION
**Pull request by roy-dydx**

### CODE REVIEW

File: pages/governance/slashing_a_validator.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Slashing a validator
+
+The chain supports slashing of misbehaving validators through governance vote.
+
+## Proposal Message
+
+The proposal should contain a single [MsgSlashValiator](https://github.com/dydxprotocol/v4-chain/blob/protocol/v4.0.0/proto/dydxprotocol/govplus/tx.proto#L19) message for each validator that should be slashed. See the inline comments for details on each field.
+
+**💡 Important: Both `tokens_at_infraction_height` and `slash_factor` must be set correctly, otherwise undelegations and redelegations might be slashed disproportionately to the rest of the validator's stake.**
```
Comment (ttl33) [inline pages/governance/slashing_a_validator.md:9]: Can we add explanations around how to pick these values? For example, how do you pick the right value for the height given the governance process takes a few days and etc

File: pages/governance/slashing_a_validator.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "title": "Slash a validator",
+    "summary": "We are proposing to slash this misbehaving validator for X reasons.",
+    "deposit": "10000000000000000000000adv4tnt",
+    "messages": [
+        {
+          "@type": "/dydxprotocol.govplus.MsgSlashValidator",
+          "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+          "validator_address": "dydxvalcons1z79h40nmd777scs93qjxaeak8m2cl6hpqg2rx9",
+          "slash_factor": "0.2",
+          "tokens_at_infraction_height": "500000000000000000000000",
```
Comment (ttl33) [inline pages/governance/slashing_a_validator.md:25]: To make it easier to reference in other places

```suggestion
          "tokens_at_infraction_height": "5000",
```

File: pages/governance/slashing_a_validator.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Proposal Message
+
+The proposal should contain a single [MsgSlashValiator](https://github.com/dydxprotocol/v4-chain/blob/protocol/v4.0.0/proto/dydxprotocol/govplus/tx.proto#L19) message for each validator that should be slashed. See the inline comments for details on each field.
+
+**💡 Important: Both `tokens_at_infraction_height` and `slash_factor` must be set correctly, otherwise undelegations and redelegations might be slashed disproportionately to the rest of the validator's stake.**
+
+### Example Proposal Json
+
+Below is an example proposal JSON file to propose a slashing a validator's stake by 0.2. After execution the validator will lose 20% of their `tokens_at_infraction_height`.
```
Comment (ttl33) [inline pages/governance/slashing_a_validator.md:13]: I think we should be specific about "validator's stake"

```suggestion
Below is an example proposal JSON file to propose a slashing a validator's total bonded tokens (= total stake, which includes both user delegated and self-delegated) by 0.2 at block height 5000. In other words:
* assuming that the proposal passes governance vote
* assuming that the validator has 1000 total bonded tokens at height 5000
* once the block height reaches 5000, the validator will lose 200 tokens
```

File: pages/governance/slashing_a_validator.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+}
+  ```
+
+## Submitting an Onchain Proposal
+
+Follow instructions [here](./submitting_a_proposal.md) to submit an onchain proposal.
+
+
+#### Disclaimer
+Users considering using the permissionless markets function of the dYdx v4 software are
```
Comment (ttl33) [inline pages/governance/slashing_a_validator.md:38]:
```suggestion
Users considering using the permissionless markets function of the dYdX v4 software are 
```

**Review by ttl33 — APPROVED** [review]

File: pages/governance/slashing_a_validator.md (hunk 4 above)
Comment (teddyding) [reply pages/governance/slashing_a_validator.md:38]: Why is this paragraph relevant for this page?

File: pages/governance/slashing_a_validator.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+* if there are unbonding undelegations and redelegations since height 5000, they will lose 0.2 of their stake, and the remaining will be taken from the validator for a total of 200 tokens lost
+```json
+{
+    "title": "Slash a validator",
+    "summary": "We are proposing to slash this misbehaving validator for X reasons.",
+    "deposit": "10000000000000000000000adv4tnt",
+    "messages": [
+        {
+          "@type": "/dydxprotocol.govplus.MsgSlashValidator",
+          "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
```
Comment (ttl33) [inline pages/governance/slashing_a_validator.md:31]: would be useful to add a comment that this has to be `gov` module account

**Pull request merged**